src/forms/BenefitsTable.js

```diff
@@ -73,36 +78,41 @@ const BenefitsTable = function (props) {
         };
 
 
-
-
-
-
   const SNAPBenefitRow = function(props){
-    if (!client.current.hasSnap) {return (null);}
+    if (!client.current.hasSnap) {
+      return (null);
```

Comment:
Why have parentheses around return values?

Reply:
Not sure if I did that or if it was someone else. If it was me, I was hallucinating at the time. Maybe it was to mimic the way that Components are usually returned in parens. Not sure, though.